joomla/joomla-cms
# [4.0] installer header

Pull Request for Issue #31304

### Summary of Changes
sets correct flow for installer header


### Testing Instructions
start fresh install ( or rename configuration.php )
examine header notice logo is missing & title is offset to the right.
apply pr
run "npm ci"
examine header again in both mobile and desktop
Notice everything looks nice.


### Actual result BEFORE applying this Pull Request
![image](https://user-images.githubusercontent.com/1850089/98166369-ec252e80-1eac-11eb-80f8-381e90929f12.png)

![image](https://user-images.githubusercontent.com/1850089/98166250-bed88080-1eac-11eb-98ac-f92f828d7690.png)


### Expected result AFTER applying this Pull Request
![image](https://user-images.githubusercontent.com/1850089/98165923-4d003700-1eac-11eb-808c-41268fb8cd36.png)
![image](https://user-images.githubusercontent.com/1850089/98165967-5f7a7080-1eac-11eb-80e7-1cf6dec1e7af.png)
![image](https://user-images.githubusercontent.com/1850089/98166064-77ea8b00-1eac-11eb-8420-89236c7bf500.png)


### Documentation Changes Required
none
